### ISSUE TITLE
timelib.calendar_tuple fails on array inputs

### ISSUE THREAD

**Issue by jrs65** [skyfielders/python-skyfield#102]
I'm getting an Exception thrown when I try to use `timelib.calendar_tuple` with an array.

```python

>>> import numpy as np
>>> from skyfield import timelib                                                                                         
>>> jd_array = 2440587.5 + np.arange(10.0)                                                                                                            
>>> years, months, days, hours, minutes, seconds = timelib.calendar_tuple(jd_array)
Traceback (most recent call last):
  File "<stdin>", line 2, in <module>
  File "/bg01/homescinet/k/krs/jrs65/sfdev/python-skyfield/skyfield/timelib.py", line 575, in calendar_tuple
    whole = whole.astype(whole)
TypeError: data type not understood
```

I think the issue is that the offending line ([skyfield/timelib.py:575](https://github.com/skyfielders/python-skyfield/blob/176fdef7df325defc6cfa789e65261b71b84eafb/skyfield/timelib.py#L571)) has a slight typo and that it should be changed from:
```python
whole = whole.astype(whole)
```
to
```python
whole = whole.astype(int)
```

Making that change allows it to work correctly on array inputs, and doesn't cause any regressions in the unit tests.

I'll try and put together a PR for it later on today.

**Comment by jrs65** [comment]
A quick comment...

This issue crops up when calling `Time.tt_calendar()` or `Time.tai_calendar()`, e.g.

```python
tv = ts.utc(range(2000, 2010))
tv.tt_calendar()  # Fails
```

**Comment by brandon-rhodes** [comment]
Fixed in 10d7a890958844a70bb9abce3889b973fa602926.
